Package@swift-6.swift

```diff
@@ -510,7 +538,7 @@ extension Target {
         .grpcCore,
         .grpcProtobuf
       ],
-      swiftSettings: [._swiftLanguageMode(.v5), .enableUpcomingFeature("ExistentialAny")]
+      swiftSettings: [._swiftLanguageMode(.v6), .enableUpcomingFeature("ExistentialAny")]
```

Comment:
This one doesn't have `"InternalImportsByDefault"`

Reply:
Yes, it's because there are a bunch of generated files in this module. I will enable it for this one in a follow up PR after I've changed the code generator.

Reply:
Cool, thanks for clarifying!